src/components/templates/ExternalRoom/ExternalRoom.tsx

```diff
@@ -25,80 +14,48 @@ export interface ExternalRoomProps {
   const redirectUrl = venue.zoomUrl ?? "";
 
-  const venueLogoClasses = classNames("ExternalRoom__venue-logo");
-
   useEffect(() => {
     if (!redirectUrl) return;
+
     openUrl(redirectUrl);
   }, [redirectUrl]);
 
-  const openRoomUrl = useCallback(() => openUrl(redirectUrl), [redirectUrl]);
-
   return (
-    <VenueWithOverlay venue={venue} containerClassNames="ExternalRoom">
-      <div className="ExternalRoom__container">
-        <SparkleLogoIcon />
-        {!redirectUrl && (
-          <div className="ExternalRoom__container">
-            <div className="ExternalRoom__message">
-              Venue {venue.name} should redirect to a URL, but none was set.
-            </div>
-          </div>
-        )}
-
-        {redirectUrl && (
-          <>
-            <div className="ExternalRoom__message">
-              <div>This page should automatically open</div>
-              <a
-                href={redirectUrl}
-                className="ExternalRoom__link"
-                {...getExtraLinkProps(true)}
-              >
-                {redirectUrl}
-              </a>
-
-              <div>
-                in a new tab. If you&apos;re not seeing this, try{" "}
-                <a rel="noreferrer" href={ENABLE_POPUPS_URL} target="_blank">
-                  enabling pop ups on your browser.
-                </a>
-              </div>
-            </div>
-            <div className="ExternalRoom__content">
-              <div className="ExternalRoom__venue-container">
-                <div className={venueLogoClasses} />
-
-                <div className="ExternalRoom__venue-details">
-                  <div className="ExternalRoom__venue-title">{venue.name}</div>
-
-                  <div className="ExternalRoom__venue-subtitle">
-                    {venue.config?.landingPageConfig.subtitle}
-                  </div>
-
-                  <ButtonNG variant="primary" onClick={openRoomUrl}>
-                    Enter
-                  </ButtonNG>
-                </div>
-              </div>
-
-              <div className="ExternalRoom__venue-description">
-                <RenderMarkdown
-                  text={venue.config?.landingPageConfig.description}
-                />
-              </div>
-
-              <UserList
-                containerClassName="ExternalRoom__userlist"
-                userCount={venue.recentUserCount ?? 0}
-                usersSample={venue.recentUsersSample ?? ALWAYS_EMPTY_ARRAY}
-                activity="in here"
-                hasClickableAvatars
-              />
-            </div>
-          </>
-        )}
+    <div
+      className={styles.ExternalRoom}
+      style={{
+        background: `url(${venue.config?.landingPageConfig.coverImageUrl}) center center`,
+      }}
+    >
+      <img
+        src={venue.host?.icon}
+        alt="Venue icon"
+        className={styles.VenueIcon}
+      />
+      <div className={styles.InfoContainer}>
```

Comment:
These would be nicer if they're `camelCase`, not `PascalCase` as is with the other class names. 

The class names mapping is another matter, since what is being import from `"./ExternalRoom.module.scss"` is a `Record<string,string>` not `CSSStyleDeclaration` so the name `styles` is misleading.